db/migrate/20221010090227_add_devise_two_factor_to_users.rb

```diff
@@ -0,0 +1,9 @@
+class AddDeviseTwoFactorToUsers < ActiveRecord::Migration[6.1]
+  def change
+    add_column :users, :encrypted_otp_secret, :string
+    add_column :users, :encrypted_otp_secret_iv, :string
+    add_column :users, :encrypted_otp_secret_salt, :string
+    add_column :users, :consumed_timestep, :integer
+    add_column :users, :otp_required_for_login, :boolean
```

Comment:
@ajaystha42 should this be set to `default: false`?

Reply:
@donrestarone We are handling the default true case when new user created if 2fa is already enabled. I believe we dont need to set it to false by default.

Reply:
makes sense. we need to setup false by default